lib/ark_elixir/delegate.ex

```diff
@@ -81,10 +81,11 @@ defmodule Ark_Elixir.Delegate do
         "success" => true}
     """
     def get_delegate(id) do
-      case String.length(id) do
-        66 -> Ark_Elixir.Api.get("api/delegates/get", [publicKey: id])
-        _ -> Ark_Elixir.Api.get("api/delegates/get", [username: id])
+      id_type = case String.length(id) do
```

Comment:
This variable name does not follow your snake case convention. I know that the unofficial Elixir style guide says to use snake case, but I follow what the Elixir source code has with using _ to indicate multiple words. This can be easily changed if needed; I wasn't able to find guidelines on what style to use.